src/nni_manager/training_service/remote_machine/gpuScheduler.ts

```diff
@@ -28,20 +28,27 @@ import {
     parseGpuIndices, RemoteMachineMeta, RemoteMachineScheduleResult, RemoteMachineTrialJobDetail, ScheduleResultType, SSHClientManager
 } from './remoteMachineData';
 
+type SCHEDULE_POLICY_NAME = 'random' | 'round-robin';
```

Comment:
Can users change schedule policy?

Reply:
No for now,  there is no need to change it from round-robin to random for now.